organization/_base.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   'parser': 'babel-eslint',
   'plugins': [
     'mocha',
+    'flow-vars',
```

Comment:
This is fine for an organization setting since it has no effect if flow is not used.